ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -540,3 +568,33 @@ message TimedOut {
 message AlreadyExists {
   string details = 1;
 }
+
+// V2
+
+message ValidationFailure {
+    string details = 1;
```

Comment:
> Looks good! We should consider some alternatives for changing `daml_kvutils.proto` so that the schema is easier to maintain.

I would split it a bit into multiple files tbh. All the rejections code, for example, it would make a lot more sense to have it in its own file.

Reply:
Digging through IJ to fix the formatter that I use as it's not the first time it causes issues for me